pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.401</version>
+        <version>2.21</version>
     </parent>
 
     <artifactId>versioncolumn</artifactId>
```

Comment:
NIT: you should comment on the line above, not below. Because if not Github shows the line above, not below making reviewing reviews not very easy :yum: 

Going to try something, any great idea/proposal?